catalyst/callbacks/checkpoint.py

```diff
@@ -568,11 +568,13 @@ def on_epoch_end(self, runner: "IRunner") -> None:
                 dict(runner.epoch_metrics),
             )
             self.top_best_metrics.append(metrics_record)
-            # truncate checkpoints
-            self._truncate_checkpoints()
-            # save checkpoint metrics
-            metrics_log = self._prepare_metrics_log(float(score), dict(runner.epoch_metrics))
-            save_config(metrics_log, f"{self.logdir}/{self.metrics_filename}")
+            # do operations only from master process
+            if runner.engine.is_master_process:
```

Comment:
Checkpoint callback should be `master`-node only, isn't it?

Reply:
Yeah